integration/getServices/client_get_services_test.go

```diff
@@ -0,0 +1,74 @@
+package main
```

Comment:
any reason this can't go under the integration dir and change package to integration?

Reply:
👍  fixed